diff -urN ../bkp/aruco/utils/aruco_create_board.cpp utils/aruco_create_board.cpp 
--- ../bkp/aruco/utils/aruco_create_board.cpp   2015-06-24 16:33:43.000000000 +0200
+++ utils/aruco_create_board.cpp        2019-12-12 16:57:05.726560461 +0100
@@ -27,6 +27,7 @@
 ********************************/
 #include "board.h"
 #include <cstdio>
+#include <sstream>
 #include <opencv2/highgui/highgui.hpp>
 #include "arucofidmarkers.h"
 using namespace std;
@@ -50,8 +51,11 @@
             pixSize = atoi(argv[4]);
         if (argc >= 6)
             typeBoard = atoi(argv[5]);
-        if (argc >= 7)
-            interMarkerDistance = atof(argv[6]);
+        if (argc >= 7) {
+            std::stringstream ss;
+            ss << argv[6];
+            ss >> interMarkerDistance;
+        }
         if ((interMarkerDistance > 1.f) || (interMarkerDistance < 0.f)) {
             cerr << "Incorrect interMarkerDistance '" << interMarkerDistance << "' -- needs to be [0,1]" << endl;
             return -1;
